docs/loaders/apirefloader.fsx

```diff
@@ -38,7 +38,7 @@ let loader (projectRoot: string) (siteContet: SiteContents) =
           Path.Combine (projectRoot, "..", "build")
         ]
       for (label, dll) in dlls do
-        let output = MetadataFormat.Generate(dll, markDownComments = true, publicOnly = true, libDirs = libs)
+        let output = MetadataFormat.Generate(dllFile = dll, markDownComments = true, publicOnly = true, libDirs = libs, parameters = [("project-name", label)])
```

Comment:
This is a go at fixing the 'no project name specified' error at https://github.com/Numpsy/FSharpLint/actions/runs/16079166194/job/45380884425#step:7:58